backend/blueprint/api/.env.example

```diff
@@ -0,0 +1,22 @@
+# Genral Settings
+SECRET = ""
+APP_SETTINGS = "development|testing|staging|production"
+
+DEBUG = True
+
+# Postgresql Configurations
+POSTGRES_USER = '' 
+POSTGRES_PASSWORD = '' 
+POSTGRES_HOST = '' 
+POSTGRES_PORT = '' 
+POSTGRES_DATABASE = '' 
+POSTGRES_SECRET = ''
+
+# Mail Configurations
```

Comment:
Why we need this?

Reply:
Because of this https://github.com/fossasia/badgeyay/pull/712

Reply:
The whole motive is to store the configuration at one place so that user don't have to move around to check where the problem is